charts/velero/values.yaml

```diff
@@ -302,6 +302,10 @@ configuration:
   disableControllers:
   # `velero server` default: 1m
   storeValidationFrequency:
+  # `velero server` default: 1h
+  garbageCollectionFrequency:
+  # `velero server` default: []
+  extraArgs: []
```

Comment:
Providing extraArgs is brilliant.
But I am thinking that should we always keep helm chart args aligned with current velero server-supported arguments?

Also, if we want to support `extraArgs`, add `garbageCollectionFrequency` reasonable?

Reply:
Hi @jenting !
I'd say it depends on the architecture. Since you guys were following that pattern I decided to keep that way while still adding the extraArgs as a backup plan to prevent other missing values in the future.

If you think it's better that way I can remove the garbageCollectionFrequency part. It also sounds good for me.

Reply:
🤔 I don't have a strong opinion on this. But I'd prefer to use garbageCollectionFrequency part rather than extraArgs.

WDYT? @ywk253100 @reasonerjt

Reply:
I would vote for the `garbageCollectionFrequency` approach, thanks @jenting .

Reply:
If you guys wanna check another repo that uses this approach you can check [cert-manager](https://github.com/cert-manager/cert-manager/blob/master/deploy/charts/cert-manager/values.yaml#L109).
But any case you guys decide will address my issue. Let me know what you decide so I can make the necessary changes tomorrow. (3am here)

Reply:
Hey @jenting, I just corrected the PR to follow the pattern you guys prefered.
Please tell me if there's anything else I should do.